instrumentation/jms-1.1/src/jms2Test/groovy/SpringTemplateJms2Test.groovy

```diff
@@ -119,16 +119,16 @@ class SpringTemplateJms2Test extends AgentTestRunner {
     receivedMessage.text == "responded!"
     assertTraces(4) {
       trace(0, 1) {
-        producerSpan(it, 0, destinationType, destinationName)
+        consumerSpan(it, 0, destinationType, destinationName, msgId.get(), null, "receive")
```

Comment:
I didn't add a `sleep` after `Thread.start` but maybe should, will see if this is flaky